scripts/utils.js

```diff
@@ -0,0 +1,258 @@
+const hre = require("hardhat");
+
+const witnet = require("../witnet/assets")
+const addresses = witnet.getAddresses(hre.network.name)
+const utils = require("witnet-solidity/utils")
+
+module.exports = {
+    camelize,
+    extractArtifactConstructorArgsTypes,
+    extractErc2362CaptionFromKey,
+    extractRequestKeyFromErc2362Caption,
+    extractRouteKeyFromErc2362Caption,
+    flattenWitnetArtifacts: utils.flattenWitnetArtifacts,
+    getWitnetPriceFeedsContract,
+    getWitnetPriceRouteSolverContract,
+    getWitnetRequestContract,
+    getWitnetRequestResultDataTypeString,
+    getWitnetResultStatusString,
+    isDryRun: utils.isDryRun,
+    isNullAddress: utils.isNullAddress,
+    numberWithCommas,
+    overwriteJsonFile: utils.overwriteJsonFile,
+    readJsonFromFile: utils.readJsonFromFile,
+    secondsToTime,
+    traceHeader: utils.traceHeader,
+    traceTx,
+    traceWitnetPriceFeed,
+    traceWitnetPriceRoute,
+}
+
+function camelize(str) {
```

Comment:
If you prefer a npm package, [camelcase](https://www.npmjs.com/package/camelcase) and [lodash.camelcase](https://www.npmjs.com/package/lodash.camelcase) are widely used.

Reply:
I'd rather avoid adding one more dependency to the pacakge. In fact, this PR includes changes for getting rid of a former dependency to lodash.merge